be/Cf.Application/Services/ImagesCloudServices/ImageUploadService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Cf.Application.Services.Interfaces;
+using Cf.Domain.Exceptions;
+using Cf.Domain.Exceptions.Messages;
+using CloudinaryDotNet;
+using CloudinaryDotNet.Actions;
+using Microsoft.Extensions.Configuration;
+
+namespace Cf.Application.Services.ImageUploadServices;
+
+public class ImageUploadService : IImageUploadService
+{
+    private readonly Cloudinary _cloudinary;
+
+    public ImageUploadService(IConfiguration configuration)
+    {
+        Account account = new(
+            configuration["Cloudinary:CloudName"],
+            configuration["Cloudinary:ApiKey"],
+            configuration["Cloudinary:ApiSecret"]
+        );
+        _cloudinary = new Cloudinary(account);
+    }
+
+    public async Task<string> Upload(string imageBase64)
+    {
+        try
+        {
+            byte[] imageBytes = Convert.FromBase64String(imageBase64);
+
+            using MemoryStream stream = new(imageBytes);
+            ImageUploadParams uploadParams = new()
+            {
+                File = new FileDescription(Guid.NewGuid().ToString(), stream)
+            };
+
+            ImageUploadResult uploadResult = await _cloudinary.UploadAsync(uploadParams);
```

Comment:
var uploadResult

Reply:
isn't imageuploadversult more specific? I believe we should use non generic types like var where possible.

Reply:
Visual studio provides these types when you hover over the variable. No need to be specific in every line ;)